src/main/java/module_two_1/models/enums/Messenger.java

```diff
@@ -0,0 +1,21 @@
+package module_two_1.models.enums;
+
+public enum Messenger {
+    NULL_FACULTY("Отсутствие факультетов в университете"), NULL_GROUP("Отсутствие групп на факультете"),
+    NULL_STUDENT("Отсутствие студентов в группе"), NULL_DISCIPLINE("Отсутсвие предметов у студента"),
+    ILLEGAL_MARKS("Оценка ниже 0 или выше 10"),
+    COUNT_BY_STUDENT("Cредний балл по всем предметам студента"),
+    COUNT_BY_ALL_UNIVERSITY("Средний балл для всего университета по предмету"),
+    COUNT_BY_CONDITION("Cредний балл по (предмету,группа,факультет :");
+
+    private String mess;
```

Comment:
This variable name is quite confusing. Consider using more informative names please

Reply:
Fixed